c10/core/DispatchKey.h

```diff
@@ -59,6 +59,8 @@ enum class DispatchKey : uint8_t {
 
   VariableTensorId,
 
+  ProfilerId,
```

Comment:
This is the nittiest of nits but I think the old convention of appending "Id" on every dispatch key doesn't make sense anymore because it's not TensorTypeId anymore, it's DispatchKey, so we should just drop the suffix and call it straight "DispatchKey::Profiler"